vscode-extension/package.json

```diff
@@ -140,7 +140,7 @@
     "@material-ui/icons": "4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
     "@xstate/react": "1.6.3",
-    "lexical": "0.8.1",
+    "lexical": "0.15.0",
```

Comment:
Isn't this file missing an explicit dependency to `@lexical/react`? Shouldn't there be an update of the package.json of `sirius-web` and `sirius-web-application`?